vic/drivers/classic/src/alloc_veg_hist.c

```diff
@@ -38,29 +38,27 @@ alloc_veg_hist(int                nrecs,
 {
     int i, j;
 
-    (*veg_hist) = (veg_hist_struct **) calloc(nrecs, sizeof(veg_hist_struct *));
+    (*veg_hist) = calloc(nrecs, sizeof(*(*veg_hist)));
```

Comment:
I think that is correct. I would recommend leaving the extra parentheses off on both sides, i.e. 

`*veg_hist = calloc(nrecs, sizeof(**veg_hist));`

would be my preference